docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,94 @@ which groups these values were observed, as well as the difference and ratio
 between the maximum and the minimum values. For more information refer to the
 subpackage :code:`fairlearn.metrics`.
 
+Failure modes encountered when abstracting
+--------------------------------------------------
+
+In "Fairness and Abstraction in Sociotechnical Systems" Selbst et al. [#4]_
+identify failure modes that arise from applying abstraction. They identify
+them as:
+
+* *The Framing Trap*
+
+* *The Portability Trap*
+
+* *The Formalism Trap*
+
+* *The Ripple Effect Trap*
+
+* *The Solutionism Trap*
+
+These traps are inspired by the observation that each of these traps are the
+result of failing to consider the way in with social context and technology
+are interrelated, as well as a deeper understanding of "the social" in order to
+solve problems Ackerman [#5]_.
+
+The Framing Trap
+^^^^^^^^^^^^^^^^
+
+Selbst et al. [#4]_ define this as a "failure to model the entire system over
+which a social criterion, such as fairness, will be enforced".
+
+It is the practice of failing to consider the full picture surrounding sources of
+bias.
```

Comment:
Ah, we recommend not using the term bias (since it's overloaded and means different things to different communities), and instead talking about fairness issues or fairness-related harms (see contributor guide section [here](https://fairlearn.org/v0.6.2/contributor_guide/how_to_talk_about_fairness.html))

Reply:
Removed word "bias" from definitions and examples, substituted into "social context", "social construct", or other context/social related sources of understanding.